### DISCUSSION TITLE
Disabled TOC still takes up room?

### DISCUSSION THREAD

**Discussion by iainhallam** [combostrap/combo#66] [General]
Hi; just want to check if this is right. I've disabled the TOC by setting the maximum level to 0 in the configuration, but on internal pages using the "holy" template, there's still an empty column where the TOC could go. Is there a template that has the sidebar but no TOC? Should the holy template detect whether there's a TOC and use the space for content instead? Thanks!

**Comment by gerardnico** [comment]
The index template:

https://combostrap.com/template/index-template-zbwtuq78

**Reply by gerardnico** [reply]
The toc is part of the template. You delete it by choosing a template without toc.

https://combostrap.com/layout-elements/combostrap-ui-table-of-content-toc-gn27tket#how_to_add_or_delete_a_toc

**Reply by iainhallam** [reply]
Clearly doing this too late at night - thanks for the pointers!
